src/components/twilio-voice-basic-call-control/routes.js

```diff
@@ -19,4 +48,20 @@ router.post('/conferences/:conferenceSid/participants/:callSid', async (req, res
   res.sendStatus(200);
 });
 
+router.post('/conferences/:conferenceSid/participants/add/:recipient', async (req, res) => {
```

Comment:
For a better design and for this to be considered restful, rename this to `/conferences/:conferenceSid/participants` then send the new participant in the post body. Let's call the parameter in the post body `to`.